### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'isKeyObject')

### DISCUSSION THREAD

**Discussion by mglikesbikes** [panva/jose#399] [Q&A]
### What happened?

Using Remix JS + Stytch so want to preface this by saying, very much doubt this is this library's problem; just looking for help.

Getting an error when compiling a cloudflare worker with `wrangler@beta`. This is the raw `function/[[path]].js` that's compiled _from_ Remix in preparation for Workers to use it:

```// node_modules/jose/dist/node/cjs/runtime/is_key_object.js
var require_is_key_object = __commonJS({
  "node_modules/jose/dist/node/cjs/runtime/is_key_object.js"(exports) {
    "use strict";
    init_react();
    Object.defineProperty(exports, "__esModule", { value: true });
    var crypto_1 = require_crypto2();
    var util = require_util();
    console.log(">>>>????", util.types);
    exports.default = util.types.isKeyObject
      ? (obj) => util.types.isKeyObject(obj)
      : (obj) => obj != null && obj instanceof crypto_1.KeyObject;
  },
});
```

`util.types` is not defined so the whole thing's blowing up. Imported stytch using:

```
import * as stytch from "stytch";
```

In a typescript project.

Thanks in advance for any help…

### Version

4.6.0

### Runtime

Cloudflare Workers

### Runtime Details

❯ wrangler --version 0.0.26

### Code to reproduce

```js
import * as stytch from "stytch";

export const loader: LoaderFunction = async ({ request, context, params }) => {
  const client = new stytch.Client({
    project_id: context.STYTCH_PROJECT_ID_TEST,
    secret: context.STYTCH_SECRET_TEST,
    env: stytch.envs.test,
  });

  …

  return json({ … });
};
```

### Required

- [X] I have searched the issues tracker and discussions for similar topics and couldn't find anything related.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/panva/jose/blob/main/CODE_OF_CONDUCT.md)

**Comment by panva** [comment]
CF Workers is not a node runtime. Whatever bundler you use must be configured to make use of a browser or worker bundle.

**Reply by trev-gulls** [reply]
How do I configure this for tests using jest and node runtime?
